bin/copy-assets.js

```diff
@@ -19,50 +19,52 @@ function start() {
   console.log("start: copy assets")
   const projectPath = path.resolve(__dirname, "..")
   const mcModulePath =  "devtools/client/debugger/new";
-  const mcPath = args.mc ? args.mc : feature.getValue("firefox.mcPath");
+  let mcPath = args.mc ? args.mc : feature.getValue("firefox.mcPath");
+
+  // resolving against the project path in case it's relative. If it's absolute
+  // it will override whatever is in projectPath.
+  mcPath = path.resolve(projectPath, mcPath);
```

Comment:
projectPath is always set as the working directory in the rest of the script, so I'm not sure if this is really needed.
I tried commenting out this line and the script still ran correctly for both in and out of tree mc clones. Am I missing something? (in any case it doesn't hurt to have it)